VK_App/VK_App/Information/NewsInformation/NewsItems.swift

```diff
@@ -0,0 +1,14 @@
+// NewsItems.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Список новостей
+struct NewsItems: Decodable {
+    /// список новостей
```

Comment:
```suggestion
    /// Список новостей
```
Так же все с большой